cachi2/core/package_managers/gomod.py

```diff
@@ -3,7 +3,7 @@
 import os
 import re
 import shutil
-import subprocess  # nosec
+import subprocess
```

Comment:
**Commit message**:
Can we add a very brief decision on why we're disabling them? This is solely to have that bit of info tracked in the git history.

Reply:
> Can we add a very brief decision on why we're disabling them? This is solely to have that bit of info tracked in the git history.

Scratch this, hit send too soon. The only change I'd make is:

_s/The commit adds the options:/It's better to explicitly list all disabled rules and so this patch disables:_